.github/workflows/codeception.yml

```
@@ -29,8 +29,8 @@ jobs:
         strategy:
             matrix:
                 include:
-                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: highest, experimental: false }
-                    - { php-version: 8.2, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: highest, experimental: false }
+                    - { php-version: 8.1, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: lowest, pimcore_version: "", experimental: false }
```

what about 
```suggestion
                    - { php-version: 8.1, database: "mariadb:10.3", server_version: "xyz", dependencies: lowest, pimcore_version: "", experimental: false }
```

since the minimum in the Core is 10.3?
https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Requirements/#database-server

Probably would have helped in catching this issues
https://github.com/pimcore/web2print-tools/issues/78#issuecomment-1596683953